src/ProjectSystemTools/BuildLogging/Model/Frontend/IFrontendBuildTableDataSource.cs

```diff
@@ -1,22 +1,21 @@
 ﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
 
-using Microsoft.Build.Framework;
 using Microsoft.VisualStudio.Shell.TableManager;
 
-namespace Microsoft.VisualStudio.ProjectSystem.Tools.Providers
+namespace Microsoft.VisualStudio.ProjectSystem.Tools.BuildLogging.Model.Frontend
 {
-    public interface IBuildTableDataSource : ITableDataSource
+    public interface IFrontendBuildTableDataSource : ITableDataSource
     {
         ITableManager Manager { get; set; }
 
-        bool IsLogging { get; }
+        bool IsLogging();
 
         void Start();
 
         void Stop();
 
         void Clear();
 
-        ILogger CreateLogger(bool isDesignTime);
+        string RetrieveLogForBuild(int buildID);
```

Comment:
The more common verb in .NET APIs tends to be `Get`. You may also like to consider what happens if the value is not found, and make it easier for the caller to handle that case by using something like `bool TryGetLog(int buildId, out string log)`.